tslint.json

```diff
@@ -20,7 +20,7 @@
     "no-implicit-dependencies": [
       true,
       "dev",
-      ["@datadog/browser-logs", "@datadog/browser-rum", "@datadog/browser-core"]
+      ["@datadog/browser-logs", "@datadog/browser-rum", "@datadog/browser-core", "ajv"]
```

Comment:
FMU, `ajv` is installed because it is listed in the `rum` workspace dependencies, and hoisted in the root `node_modules` folder. Why not listing `ajv` as a root package dependency? We could either keep or remove it from `rum` dependencies.